tests/test_flows.py

```diff
@@ -2416,6 +2441,159 @@ def my_flow():
         assert my_mock.call_args_list == [call(), call()]
 
 
+class TestFlowHooksOnCancellation:
+    def test_on_cancellation_hooks_run_on_cancelled_state(self):
+        my_mock = MagicMock()
+
+        def cancelled_hook1(flow, flow_run, state):
+            my_mock("cancelled_hook1")
+
+        def cancelled_hook2(flow, flow_run, state):
+            my_mock("cancelled_hook2")
+
+        @flow(on_cancellation=[cancelled_hook1, cancelled_hook2])
+        def my_flow():
+            return State(type=StateType.CANCELLING)
+
+        my_flow._run()
+        assert my_mock.mock_calls == [call("cancelled_hook1"), call("cancelled_hook2")]
```

Comment:
I'm sure this was a pattern established before you wrote a hook, but there's no coverage of the items the hook is called with here. If you just used two mocks directly as your hooks instead of calling them manually from within the hook you could use `assert_called_once_with(...)` and actually confirm the correct data is being passed 🤷‍♂️ 

No need to hold up this pull request though.